frontend/pages/policies/ManagePoliciesPage/components/CalendarEventsModal/CalendarEventsModal.tsx

```diff
@@ -133,28 +111,14 @@ const CalendarEventsModal = ({
     setFormData(newFormData);
   };
 
-  const onPolicyEnabledChange = useCallback(
-    (newVal: { name: FormNames; value: boolean }) => {
-      const { name, value } = newVal;
-      const newFormPolicies = formData.policies.map((formPolicy) => {
-        if (formPolicy.name === name) {
-          return { ...formPolicy, isChecked: value };
-        }
-        return formPolicy;
-      });
-      const newFormData = { ...formData, policies: newFormPolicies };
-      setFormData(newFormData);
-    },
-    [formData]
-  );
```

Comment:
This is now handled by the `PoliciesPaginatedList` and `PaginatedList` components.